src/interface.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import re
+from pathlib import Path
+
+from src.utils import read_csv_file, read_excel_file, read_json_file
+from src.widget import get_date, mask_account_card
+
+root_path = Path(__file__).resolve().parents[1]
```

Comment:
Правильнее два раза вызвать родителя
```suggestion
root_path = Path(__file__).resolve().parent.parent
```
Причем я бы создал в src модуль `config.py` и там определил эту переменную - чтоб переиспользовать в других местах